Documentation/ConfigFile.md

```diff
@@ -1,10 +1,17 @@
 # SwiftGen Configuration File
 
-In order to avoid invoking SwiftGen manually multiple times — one for each subcommand — and having to remember each arguments to pass every time, you can instead use a YAML configuration file to configure everything.
+SwiftGen expects you to have a `swiftgen.yml` configuration file at the root of your repository, to specify which files to parse, what template to use, and where to write the generated files.
 
-Simply create a YAML file named `swiftgen.yml` at the root of your repository with the structure described below, then just run `swiftgen` (without any argument).
+## Creating a Config file
 
-> For more options (using a different file, checking your config file for errors…), see further below.
+To create your configuration file, you can use the **`swiftgen config init`** command.
+
+This will create (and open) an example configuration file that you can use for inspiration, with some typical entries commented out. All you have to do is to then uncomment the parts of the config that you need, adapt them to your project, and/or copy&paste the examples to add configuration for additional parsers.
+
+You can of course also create the `swiftgen.yml` configuration file manually from scratch, using the text editor of your choice. The file is just regular a YAML _(note: JSON, being a subset of YAML, is also supported)_
```

Comment:
I think that the last sentence is missing some bits.

Reply:
It isn't really, but I can try to elaborate.

```suggestion
You can of course also create the `swiftgen.yml` configuration file manually from scratch, using the text editor of your choice. The file just has to be in YAML format _(note: JSON, being a subset of YAML, is also supported)_ and provide the keys described below.
```

Reply:
Yeah that's better. The initial "The file is just regular a YAML" was just incomplete I think.